### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by emmalearnscode** [imsaptarshi/HacktoSearch#8]
Some of your elements are missing aria roles and labels. Is it ok if I make some accessibility improvements to the site?

**Comment by imsaptarshi** [comment]
> Some of your elements are missing aria roles and labels. Is it ok if I make some accessibility improvements to the site?

Great!
Go ahead with it 😉
